grails-app/domain/org/pih/warehouse/invoice/InvoiceItemType.groovy

```diff
@@ -0,0 +1,18 @@
+package org.pih.warehouse.invoice
+
+enum InvoiceItemType {
```

Comment:
If we're keeping this (I assumed we wouldn't) we should call this InvoiceItemTypeCode to leave room for adding a proper domain in the future.

Reply:
@jmiranda no, were removing this, the PR was not updated to the most recent changes yet